File: elixir/7ku/highest_and_lowest.ex
defmodule Kata do
  def high_and_low(s) do
    String.split(s)
    |> Enum.map(&String.to_integer/1)
    |> Enum.min_max()
    |> Tuple.to_list()
    |> Enum.sort(:desc)
    |> Enum.join(" ")
  end
end

File: ruby/7ku/Highest and Lowest.rb
#Highest and Lowest
# https://www.codewars.com/kata/554b4ac871d6813a03000035

def high_and_low(numbers)
  numbers.split.map(&:to_i).minmax.reverse.join(' ')
end

p high_and_low("8 3 -5 42 -1 0 0 -9 4 7 4 -4") # "42 -9"